### PR TITLE
fix: add solution to binay agent guide article

### PR DESCRIPTION
**Pull request by sanghyundev**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Review by thecodingaviator — APPROVED** [review]
LGTM!

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/binary-agents/index.md [hunk 1]
```diff
@@ -147,6 +147,29 @@ Make sure that each time you transcode a character from binary to decimal, you r
...
+```javascript
+function binaryAgent(str) {
+  const re = /\d{8}\s*/g;
+  return str.replace(re, bin => String.fromCharCode(parseInt(bin, 2)));
+}
+
+// test here
+binaryAgent("01000001 01110010 01100101 01101110 00100111 01110100 00100000 01100010 01101111 01101110 01100110 01101001 01110010 01100101 01110011 00100000 01100110 01110101 01101110 00100001 00111111");
+```
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/repls/PointlessUltimateAddition' target='_blank' rel='nofollow'>Run Code</a>
```
Comment (thecodingaviator) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/binary-agents/index.md]:
```suggestion
```
Hi, thank you for your PR but now we are no longer accepting links or other resources (images, etc.) which are not under our control, i.e. which a user or service might remove and is not under our control.

Please remove this line

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@thecodingaviator This PR has gone stale.

**Comment by RandellDawson** [comment]
Thank you for opening this pull request.

This is a standard message notifying you that we’ve reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

**Reason:** Since your PR was created, the master branch content has updated making some of your changes unnecessary or need to be reorganized to fit the current version.  Please re-sync your fork so you are looking at the latest version of master and then create a new PR if you find any changes warranted.  

Thank you and happy coding.

**Pull request closed without merging**